.github/workflows/ci.yml

```diff
@@ -76,7 +76,7 @@ jobs:
     needs: [core]
     strategy:
       matrix:
-        java: [ '8', '15' ]
+        java: [ '8', '15', '16' ]
```

Comment:
When modifying these matrix builds, please make sure to keep the total up to date: https://github.com/guardrail-dev/guardrail/blob/master/codecov.yml#L4-L11

Reply:
This is a workaround for what seems to be a bug in codecov where it just fails the build and doesn't recover even when subsequent submissions come in.

Reply:
thank you